### PR TITLE
Pikalax's Easy chat

### PR DESCRIPTION
**Pull request by DizzyEggg**
#125 can be closed down.

### CODE REVIEW

File: include/global.h [hunk 1]
```diff
@@ -719,8 +719,8 @@ struct LilycoveLadyQuiz
     /*0x000*/ u8 id;
     /*0x001*/ u8 phase;
     /*0x002*/ u16 unk_002[9];
-    /*0x014*/ u16 unk_014;
-    /*0x016*/ u16 unk_016;
+    /*0x014*/ u16 unk_014[1];
```
Comment (Diegoisawesome) [inline include/global.h]: Is this not unk_014[2], perhaps? This is just bizarre

File: src/easy_chat.c [hunk 2]
```diff
@@ -0,0 +1,810 @@
...
+{
+    u16 v0;
+    s16 *data;
+
+    data = gTasks[taskId].data;
+    switch (data[EZCHAT_TASK_STATE])
+    {
+    case 0:
+        SetVBlankCallback(sub_811A290);
+        BlendPalettes(-1, 16, 0);
```
Comment (Diegoisawesome) [inline src/easy_chat.c]: Use 0xFFFFFFFFF to show it's a bitmask

File: src/easy_chat.c [hunk 3]
```diff
@@ -0,0 +1,810 @@
...
+    }
+    gUnknown_0203A118->unk_0d = (sub_811F3AC() - 1) / 2 + 1;
+    return TRUE;
+}
+
+static void EasyChat_FreeResources(void)
+{
+    if (gUnknown_0203A118 != NULL)
+    {
+        free(gUnknown_0203A118);
```
Comment (Diegoisawesome) [inline src/easy_chat.c]: FREE_AND_SET_NULL macro

File: include/global.h (hunk 1 above)
Comment (DizzyEggg) [reply include/global.h]: Don't ask me, it's Pikalax's.

Comment (Diegoisawesome) [reply include/global.h]: Try making the change and see if it works

Comment (DizzyEggg) [reply include/global.h]: I just made it two seperate u16, like it was before.

**Pull request merged**